src/ResourceManager/ApiManagement/Commands.ApiManagement/Commands.ApiManagement.csproj

```diff
@@ -65,10 +65,22 @@
     <Reference Include="Microsoft.Data.Services.Client, Version=5.8.1.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35, processorArchitecture=MSIL">
       <HintPath>..\..\..\packages\Microsoft.Data.Services.Client.5.8.2\lib\net40\Microsoft.Data.Services.Client.dll</HintPath>
     </Reference>
+    <Reference Include="Microsoft.Rest.ClientRuntime, Version=2.0.0.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35, processorArchitecture=MSIL">
```

Comment:
Remove Microsoft.Rest.ClientRuntime and Microsoft.Rest.ClientRuntime.Azure, as these are imported by common dependencies.